### PR TITLE
MF-8:Add module extension points to master-single-page-application.jsp

### PR DESCRIPTION
**Pull request by enyachoke**
Added extension points for custom html in the head and body.
https://issues.openmrs.org/browse/MF-8

### CODE REVIEW

File: omod/src/main/webapp/master-single-page-application.jsp [hunk 1]
```diff
@@ -21,7 +22,9 @@
       window.getOpenmrsSpaBase = function() { return window.openmrsBase + window.spaBase + '/';};
       System.import("@openmrs/root-config");
     </script>
+      <c:import url="${requestScope.spaHeadContentUrl}" />
```
Comment (internettrans) [inline omod/src/main/webapp/master-single-page-application.jsp]: Indentation here is off

File: omod/src/main/webapp/master-single-page-application.jsp [hunk 2]
```diff
@@ -21,7 +22,9 @@
...
   </head>
   <body>
+  <c:import url="${requestScope.spaBodyContentUrl}" />
```
Comment (internettrans) [inline omod/src/main/webapp/master-single-page-application.jsp]: indentation here also looks off

File: omod/src/main/java/org/openmrs/module/util/SingleSpaConstants.java [hunk 3]
```diff
@@ -20,6 +20,10 @@
 	
 	public static final String GP_KEY_SPA_BASE_URL = "spa.baseUrl";
 
+    public static final String GP_BODY_CONTENT_URL = "spa.bodyContentUrl";
```
Comment (internettrans) [inline omod/src/main/java/org/openmrs/module/util/SingleSpaConstants.java]: Please use tabs instead of spaces in this file to match the other code here.

**Review by internettrans — CHANGES_REQUESTED** [review]
This looks good to me. Just nitpicky indentation stuff to fix and then let's merge.

File: omod/src/main/webapp/master-single-page-application.jsp (hunk 1 above)
Comment (enyachoke) [reply omod/src/main/webapp/master-single-page-application.jsp]: I have made the change.

File: omod/src/main/java/org/openmrs/module/util/SingleSpaConstants.java (hunk 3 above)
Comment (enyachoke) [reply omod/src/main/java/org/openmrs/module/util/SingleSpaConstants.java]: This should be okay now.

**Review by internettrans — APPROVED** [review]

File: omod/src/main/java/org/openmrs/module/spa/servlet/SpaServlet.java [hunk 4]
```diff
@@ -20,8 +20,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import static org.openmrs.module.util.SingleSpaConstants.DEFAULT_SPA_BASE_URL;
-import static org.openmrs.module.util.SingleSpaConstants.GP_KEY_SPA_BASE_URL;
+import static org.openmrs.module.util.SingleSpaConstants.*;
```
Comment (dkayiwa) [inline omod/src/main/java/org/openmrs/module/spa/servlet/SpaServlet.java:23]: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

**Pull request merged**